internal/observability/event/telemetry_filter.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package event
+
+import (
+	"errors"
+	"reflect"
+
+	"google.golang.org/protobuf/proto"
+)
+
+// protoFilter is a signature for a struct field validation test
+type protoFilter func(field reflect.StructField) bool
+
+// telemetryFilter checks a struct field should be included in observation telemetry data
+func telemetryFilter(field reflect.StructField) bool {
+	if field.Tag.Get("eventstream") == "observation" {
+		// log.Printf("%s is allowed by Tags (%s:\"%s\")", name, tag, tagValue)
```

Comment:
I assume this was left in here by mistake?

Reply:
I agree with @johanbrandhorst: we should remove this commented out code.

Reply:
Yes it was. It's been removed.